File: Assets/Scripts/GunGraphics.js
var gunType : String;
var gunsound : AudioSource = GetComponent.<AudioSource>();
gunType = PickUpGun.gunName;
function Update() {
if(Input.GetButtonDown("Fire1")) {
switch (gunType){

	case "FiveSeven":

	gunsound.Play();
	GetComponent.<Animation>().Play("FiveSevenAnim");
	break;

	default:
	break;
		}
	}
}

File: Assets/Scripts/GunGraphics.cs
using UnityEngine;

public class GunGraphics : MonoBehaviour
{

    // Update is called once per frame
    void Update()
    {
        if (Input.GetButtonDown("Fire1"))
        {
            AudioSource gunsound = GetComponent<AudioSource>();
            gunsound.Play();
            GetComponent<Animation>().Play("GunShot");
        }
    }
}

